ui-v2/app/styles/base/icons/base-variables.scss

```diff
@@ -27,7 +27,7 @@ $check-circle-outline-svg: url('data:image/svg+xml;charset=UTF-8,<svg viewBox="0
 $check-plain-svg: url('data:image/svg+xml;charset=UTF-8,<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41L9 16.17z" fill="%23000"/></svg>');
 $chevron-down-2-svg: url('data:image/svg+xml;charset=UTF-8,<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z" fill="%23000"/></svg>');
 $chevron-down-svg: url('data:image/svg+xml;charset=UTF-8,<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6-1.41-1.41z" fill="%23000"/></svg>');
-$chevron-left-svg: url('data:image/svg+xml;charset=UTF-8,<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M15.705 16.885l-4.58-4.59 4.58-4.59-1.41-1.41-6 6 6 6 1.41-1.41z" fill="%23000"/></svg>');
+$chevron-left-svg: url('data:image/svg+xml;charset=UTF-8,<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M15.705 16.885L11.125 12.295L15.705 7.705L14.295 6.295L8.295 12.295L14.295 18.295L15.705 16.885Z" fill="%23000"/></svg>');
```

Comment:
This seems to be the uncompressed version of the icon, we shouldn't need to changed this the previous one was slightly more compressed and comes straight from `@hashicorp/structure-icons` - I'd probably say undo this change.

Reply:
Put back original SVG